Sources/HTTP/Serializer/ResponseSerializer.swift

```diff
@@ -0,0 +1,42 @@
+import Transport
+
+public final class ResponseSerializer<Stream: WriteableStream>: Serializer {
+    typealias StreamType = Stream
+    let stream: Stream
+    public init(_ stream: Stream) {
+        self.stream = stream
+    }
+    
+    public func serialize(_ response: Response) throws {
+        let message = try serialize(response as Message)
+        
+        let reasonPhrase = response.status.reasonPhrase.makeBytes()
+        let length = 5 // HTTP/
+            + 8 // 1.1_xxx_
+            + reasonPhrase.count
+            + 2 // \r\n
+            + message.count
+        
+        var buffer = Bytes()
+        buffer.reserveCapacity(length)
+        
+        buffer += [.H, .T, .T, .P, .forwardSlash, .one, .period, .one, .space]
```

Comment:
again, static here, but variable elsewhere in codebase, if we're doing this, we need to lock version and make it completely unsettable elsewhere.